examples/basic_mysql_connection/index.js

```diff
@@ -8,13 +8,22 @@ var connection = mysql.createConnection({
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+let query=['INSERT INTO todos(id,done) VALUES '+mysql.escape(18,0),
```

Comment:
Were you able to run this first query? I get the following error:

```
throw err; // Rethrow non-MySQL errors
        ^

TypeError: Cannot read property '0' of undefined
```

I think we talked about this in the Node class, but you should never push and PR code that doesn't run - it's called 'breaking the build'.